dev_guide/templates/templates_load.adoc

```diff
@@ -0,0 +1,236 @@
+:toc-title:
+:prewrap!:
+
+toc::[]
+{nbsp} +
+
+
+The advanced installation method 
+automatically creates a set of xref:../../dev_guide/templates/template_guide.adoc#dev-guide-templates-predef[predefined templates] 
+in the `openshift` project, which is a default project to 
+which all users have view access.
+
+Use the `oc get templates` command to see if the templates have been loaded into your {product-title} project.
+
+----
+$ oc get template
+NAME                    DESCRIPTION                                                                        PARAMETERS        OBJECTS
+...
+mariadb-ephemeral       MariaDB database service, without persistent storage. For more information ab...   7 (3 generated)   3
+mariadb-persistent      MariaDB database service, with persistent storage. For more information about...   8 (3 generated)   4
+mongodb-ephemeral       MongoDB database service, without persistent storage. For more information ab...   8 (3 generated)   3
+mongodb-persistent      MongoDB database service, with persistent storage. For more information about...   9 (3 generated)   4
+mysql-ephemeral         MySQL database service, without persistent storage. For more information abou...   8 (3 generated)   3
+mysql-persistent        MySQL database service, with persistent storage. For more information about u...   9 (3 generated)   4
+...
+----
+
+If the templates have not been loaded into your project, ask your administrator or follow the instructions below, if you have proper authority.
+
+----
+$ oc get templates
+No resources found.
+----
+
+
+[[dev-guide-templates-loading-before]]
+== Before You Begin
+
+Before you consider performing the tasks in this topic, confirm if these image streams and templates are already 
+registered in your {product-title} cluster by doing one of the following:
+
+* Log into the web console and click Add to Project.
+
+* List them for the openshift project using the CLI:
++
+----
+$ oc get templates -n openshift
+----
++
+Here, `openshift` is the default project.
+
+If you used a method other than the advanced install, or if the default image streams and templates are ever removed or changed, 
+you can follow this topic to create the default objects yourself. Otherwise, the following instructions are not necessary.
+
+include::install_config/imagestreams_templates.adoc[tag=is-templates-before-you-begin]
+
+[[dev-guide-templates-loading-prereqs]]
+== Loading the Templates
+
+If the default templates are not present in your {product-title} cluster, you can load them manually.
+
+Before you can load the default templates:
+
+- The
+xref:../../install_config/registry/index.adoc#install-config-registry-overview[integrated Docker registry]
+service must be deployed in your {product-title} installation.
+- You must be able to run the `oc create` command with
+xref:../../architecture/additional_concepts/authorization.adoc#roles[*cluster-admin* privileges],
+because they operate on the default *openshift*
```

Comment:
@bparees I took this list right out of the install docs. Do you know what _they_ means in relation to the default templates? I assume that the line should read _the default templates operate on the default *openshift* project_.

Reply:
i think in this context "they" is referring to the commands:

"you must be able to run the oc create commands with cluster-admin privileges because the commands operate on the default openshift namespace"  (normal users don't have permission to make changes to the openshift namespace)